taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True, null=True)
+
+    class Meta:
+        ordering = ("username",)

```

Comment:
According to the Python code style guide, you should define all model attributes before defining the `Meta` class. It seems like you've followed this guideline, but just wanted to point it out in case you add more attributes to the `Driver` model in the future. This is a code style issue.